.github/workflows/test.yml

```diff
@@ -145,6 +142,7 @@ jobs:
         moodle-plugin-ci phpunit --verbose --coverage-text --fail-on-warning
         moodle-plugin-ci behat --profile default
         moodle-plugin-ci behat --profile chrome
+        moodle-plugin-ci behat --profile firefox --tags="@local_ci&&~@app"
```

Comment:
I'm a little bit confused with this. How we are specifying firefox and get the `@app` tests run?

More yet... as far as me have set `MOODLE_APP: 'true'` on install... doesn't that mean that the 2 behat executions above (default and chrome) will be also running the `@app` tests (because the new added feature has `@local_ci` it will be run by default.

Not sure if I'm explaining, heh. Apart from the initial question, it's not clear for me why to specify `@app` is needed. I see it if we only want to run `@app` tests but not if we want to run all the `@local_ci` tests. And, in any case, how it's possible that they are passing with `--profile default` or `--profile firefox` profile?

Reply:
The run with firefox is doing precisely the opposite, running all the tests except the app's (notice the `~` before the `@app` tag, that means to skip the tests with the `@app` tag).

I added this line in the CI because when `MOODLE_APP` is set to true, the default browser is chrome. So without this, there wouldn't be any tests using firefox in the CI and we still want to make sure that non-app tests continue working.

Reply:
Crap, believe me I checked everything to see if, maybe, `MOODLE_APP` was getting precedence over `--profile`... and I did not see the `~`. Now it's clear, resolving this, thanks!